PostgreSQL CI

nyurik opened the pull request:
* Run postgresql as part of the CI
* Refactor postgis tests a bit to consolidate connection code

nyurik commented:
@pka could you take a look at the [build log](https://github.com/georust/geozero/pull/136/checks) of this PR - I am not certain how PG tests should be executed.

pka commented:
The CI failures are at `std::env::var("DATABASE_URL").unwrap()`. In the CI Yaml it's `_URI` instead of `_URL`!

pka commented on geozero/tests/postgis.rs:
```diff
@@ -1,16 +1,34 @@
+
+    #[cfg(feature = "with-postgis-postgres")]
+    pub fn get_conn() -> postgres::Client {
+        postgres::Client::connect(&get_db_string(), postgres::NoTls).unwrap()
```
I would prefer using
`postgres::Client::connect(&get_db_string(), postgres::NoTls).unwrap()`
instead of `get_conn()` in the examples.
The goal of the tests are also being usage examples which should be usable with minimal adaption.

The `get_pool()` function is ok, since it makes the tests quite a lot shorter.

nyurik replied:
fixed

nyurik commented:
@pka thx, fixed the issue, but now 4 unit tests don't pass. I fixed two of them (it seems now the shown values don't have the unneeded `.000000000` ?  Could you take a look again pls
